i18n/locales/en_us.json

```diff
@@ -50,6 +50,18 @@
   "duplicateThisFrameFramesListTool" :"Duplicate this frame",
   "deleteThisFrameFramesListTool": "Delete this frame",
   "loadingPiskelMessage": "Loading Piskel...",
+  "preferencesSetting": "PREFERENCES",
+  "resizeSetting": "RESIZE",
+  "resizeSettingDescriptorResizeTheDrawingArea":"Resize the drawing area",
+  "saveSetting": "SAVE",
+  "saveSettingDescriptorSaveToGallerySaveLocally": "Save to your gallery, save locally",
+  "saveSettingDescriptorExportAsFile": "export as a file",
```

Comment:
We don't want to do this because it make translation very difficult. English sentence structure doesn't match all other languages, so translating this will be very awkward. Instead, we should change piskel to remove the "<br>" from the middle of the sentences and let the text flow normally in the HTML. If this results in a div/box which is too wide, then we should be setting the CSS property `width` to limit the size.
Please rework the strings and templates which have <br/> tags in them. Also, check if I missed this in an previous pull requests as well.